apps/tlon-web/package.json

```diff
@@ -127,43 +94,30 @@
     "prosemirror-state": "~1.3.4",
     "prosemirror-transform": "~1.4.2",
     "prosemirror-view": "~1.23.13",
+    "@radix-ui/react-accordion": "^1.1.2",
+    "@radix-ui/react-popover": "^1.0.2",
```

Comment:
are we actually using these?

Reply:
we use them in Eyrie.tsx and Subscription.tsx